demo/src/store/StoreHelpers.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Helper functions to modify the store
+ */
+
+/**
+ * Helper function to add an item from the cart and set the inCart
+ * flag to true.
+ * @param {string} id: the id of the item to update
+ * @return {{type:string, id:string}} object describing the operation
+ */
+export function addOneToCart(id) {
+  return {
+    type: 'INCREMENT',
+    id: id,
+  };
+}
+
+/**
```

Comment:
Per 
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.

Reply:
Reworded